app/views/case_contacts/_case_contact.html.erb

```diff
@@ -39,17 +39,17 @@
         <%= link_to(t("button.delete"), case_contact_path(contact.id), method: :delete,
                     class: "btn btn-danger pull-right") if policy(current_user).destroy? && !contact.deleted? %>
       </div>
-      <div class="row">
+      <div class="row justify-content-start">
         <% if Pundit.policy(current_user, contact).update? %>
           <% if contact.quarter_editable? %>
             <%= render "case_contacts/followup", contact: contact, followup: contact.requested_followup %>
-            <div class="col-sm-5">
+            <div class="col col-sm-4" style="right: 10px;">
```

Comment:
`style="right: 10px;"` could you move this into one of our `.scss` files?

It's best practice to have css in a file.

see `app/javascript/src/stylesheets/`

Reply:
fJust updated this to better resolve it with bootstrap rather than CSS.

Reply:
Done. Ready for deploy pending passing checks.